.travis.yml

```diff
@@ -12,6 +12,7 @@ matrix:
     - go: 1.x
     - go: tip
   allow_failures:
+    - go: 1.x # FIXME currently golangci-lint doesn't work with go1.14
```

Comment:
Is 1.14 already released? The [release notes](https://golang.org/doc/devel/release.html) doesn't list it yet...

Reply:
nope but Travis update the version behind 1.x

https://travis-ci.com/go-acme/lego/jobs/291075495#L230

Reply:
Based on 1.12 the release will come shortly :wink:

Reply:
Yeah, both the go1.14 tag already exist (https://github.com/golang/go/tree/go1.14), and 1.14 is listed on the [download page](https://golang.org/dl/#stable).